extensions/ql-vscode/src/data-extensions-editor/external-api-usage-query.ts

```diff
@@ -27,14 +28,17 @@ export type RunQueryOptions = {
   token: CancellationToken;
 };
 
-export async function runQuery({
-  cliServer,
-  queryRunner,
-  databaseItem,
-  queryStorageDir,
-  progress,
-  token,
-}: RunQueryOptions): Promise<CoreCompletedQuery | undefined> {
+export async function runQuery(
+  queryName: keyof Omit<Query, "dependencies">,
```

Comment:
Similar to other `Pick`/`Omit` comments, can we avoid this change?

Reply:
This is somewhat different in that it was excluding the `dependencies` field from the `Query` type, ensuring you're only able to pick a query name. However, I've changed the query so we only need 1, so this parameter is unnecessary now.